README.md

```diff
@@ -1 +1,177 @@
-# angular-sharing-directive-controllers-readme
\ No newline at end of file
+# Sharing Directive Controllers
+
+## Overview
+
+Directives become even more powerful because we can communicate between their controllers. For instance, if we wanted to create a tabs component, we'd want to have our main `tabs` directive, and then directives for every tab. We can then link these together in order to create our tab picker.
+
+## Objectives
+
+- Describe the `require` property
+- Create another Directive
+- Create another Controller
+- Bind Controller to Directive
+- Import another Controller's instance
+
+## require
+
+We can require the parent controller for a certain directive by using the `require` property in our directive. We pass in a string, with the directives name that we want.
```

Comment:
Here, it would be helpful to see the HTML that we want to render first, then look at the logical way to split it into two directives. Explain that you want to have a tab list with an arbitrary number of tabs, and each tab will have different customizable properties.